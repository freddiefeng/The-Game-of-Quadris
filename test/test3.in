levelup
drop
clock
2right
drop
count
drop
6right
drop
4right
drop