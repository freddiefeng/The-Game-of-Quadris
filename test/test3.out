Level:      1
Score:      0
Hi Score:   0
----------
          
          
OO        
OO        
          
          
          
          
          
          
          
          
          
          
          
          
          
          
----------
Next:
    
    
  L 
LLL 

Level:      2
Score:      0
Hi Score:   0
----------
          
          
OO        
OO        
          
          
          
          
          
          
          
          
          
          
          
          
          
          
----------
Next:
    
    
  L 
LLL 

Level:      2
Score:      0
Hi Score:   0
----------
          
          
OO        
OO        
          
          
          
          
          
          
          
          
          
          
          
          
          
          
----------
Next:
    
    
  L 
LLL 

Level:      2
Score:      0
Hi Score:   0
----------
          
          
  L       
LLL       
          
          
          
          
          
          
          
          
          
          
          
          
OO        
OO        
----------
Next:
    
    
J   
JJJ 

Level:      2
Score:      0
Hi Score:   0
----------
          
L         
L         
LL        
          
          
          
          
          
          
          
          
          
          
          
          
OO        
OO        
----------
Next:
    
    
J   
JJJ 

Level:      2
Score:      0
Hi Score:   0
----------
          
  L       
  L       
  LL      
          
          
          
          
          
          
          
          
          
          
          
          
OO        
OO        
----------
Next:
    
    
J   
JJJ 

Level:      2
Score:      0
Hi Score:   0
----------
          
          
J         
JJJ       
          
          
          
          
          
          
          
          
          
          
          
  L       
OOL       
OOLL      
----------
Next:
    
    
    
IIII

Level:      2
Score:      0
Hi Score:   0
----------
          
 J        
 J        
JJ        
          
          
          
          
          
          
          
          
          
          
          
  L       
OOL       
OOLL      
----------
Next:
    
    
    
IIII

Level:      2
Score:      0
Hi Score:   0
----------
          
          
          
IIII      
          
          
          
          
          
          
          
          
          
 J        
 J        
JJL       
OOL       
OOLL      
----------
Next:
    
    
OO  
OO  

Level:      2
Score:      0
Hi Score:   0
----------
          
          
          
      IIII
          
          
          
          
          
          
          
          
          
 J        
 J        
JJL       
OOL       
OOLL      
----------
Next:
    
    
OO  
OO  

Level:      2
Score:      0
Hi Score:   0
----------
          
          
OO        
OO        
          
          
          
          
          
          
          
          
          
 J        
 J        
JJL       
OOL       
OOLL  IIII
----------
Next:
    
    
TTT 
 T  

Level:      2
Score:      0
Hi Score:   0
----------
          
          
    OO    
    OO    
          
          
          
          
          
          
          
          
          
 J        
 J        
JJL       
OOL       
OOLL  IIII
----------
Next:
    
    
TTT 
 T  

Level:      2
Score:     18
Hi Score:  18
----------
          
          
TTT       
 T        
          
          
          
          
          
          
          
          
          
          
 J        
 J        
JJL       
OOL OO    
----------
Next:
    
    
 SS 
SS  

